.resources/strings.json

```diff
@@ -19,130 +20,148 @@
 		"fr": "Fichiers sélectionnés",
 		"pl": "Wybrane pliki",
 		"ja": "選択されたファイル",
-		"es": "Archivos seleccionados"
+		"es": "Archivos seleccionados",
+		"de": "Ausgewählte Dateien"
 	},
 	"table.header.exif-before": {
 		"en": "# Exif Before",
 		"fr": "# Exif Avant",
 		"pl": "# Exif przed",
 		"ja": "# 処理前の Exif",
-		"es": "Exif antes"
+		"es": "Exif antes",
+		"de": "# Exif bevor"
```

Comment:
besser: # Exif vorher

Reply:
> besser: # Exif vorher

thank you, need PR is created
https://github.com/szTheory/exifcleaner/pull/88